src/main/resources/schema/ans/0.8.1/traits/trait_content_restrictions.json

```diff
@@ -8,6 +8,27 @@
     "content_code":{
       "type": "string",
       "description": "The content restriction code/level/flag associated with the ANS object"
+    },
+    "embargo": {
+      "description": "Embargo configuration to enforce publishing restrictions",
```

Comment:
I don't know the "voice" of the ANS schema, but I might write it `Embargoed content must not go live`. No strong opinion here.